dsl/Minitest.cc

```diff
@@ -11,17 +11,49 @@ using namespace std;
 
 namespace sorbet::dsl {
 
+namespace {
+unique_ptr<ast::Expression> addSigVoid(unique_ptr<ast::Expression> expr) {
+    ast::InsSeq::STATS_store stats;
+    stats.emplace_back(ast::MK::SigVoid(expr->loc, ast::MK::Hash0(expr->loc)));
+    return make_unique<ast::InsSeq>(expr->loc, std::move(stats), std::move(expr));
+}
```

Comment:
I don't really like using `InsSeq` this way, would it be better to return a `std::pair` instead?

Reply:
AFAIK there's a helper method for `Iseq` in MK